server/routes/api/dashboard.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const router = express.Router();
+let {
+    Course
+} = require('../../dbSchemas/attendanceSchema');
+
+
+/**GETS all the courses registered by a student
+ * ==========================================
+ * Example api call body:
+ * {
+	"studentId": "5f984a44da9eb32ba01d31dd"
+    }
+ */
+router.get('/getCoursesByStudent', (req, res) => {
+    let studentID = req.body.studentId;
+
+    Course.find({
+        registered_students: {
+            $in: [studentID]
+        }
+    }, function (err, courses) {
+        if (err || courses == null) {
+            console.log(err);
+            return res.status(500).send(err);
+        }
+        res.status(200).send(courses);
+    });
+});
+
+/**GETS all the courses created by a professor
+ * ==========================================
+ * Example api call body:
+ * {
+	"studentId": "5f98967d62a8bb73a0f8c046"
+    }
+ */
+router.get('/getCoursesCreatedByProfessor', (req, res) => {
+    let professorID = req.body.professorId;
+
+    Course.find({
+        professor: professorID
+    }, function (err, courses) {
+        if (err || courses == null) {
+            console.log(err);
+            return res.status(500).send(err);
+        }
+        res.status(200).send(courses);
+    });
+});
+
+/** Registers a student to his course
+ *  We give a student ID
+ *  Returns course data (name, prof name, if it's mandatory)
+ */
+router.post('/registerForCourse', (req, res) => {
+    let studentID = req.body.studentID;
+    let courseID = req.body.courseID;
```

Comment:
There seems to be inconsistencies b\t using `req.body.courseID` vs `req.body.courseId`. We should make sure to make this uniform so that we don't get too confused.

Reply:
I say this because the GET endpoints at the top use a different convention from this endpoint

Reply:
For sure, will change that.